### DISCUSSION TITLE
Add your own collection template

### DISCUSSION THREAD

**Discussion by M0TFF** [mampfes/hacs_waste_collection_schedule#2940] [Ideas]
Is it possible to have some sort of template that we could fill in so that we can add our own collection details rather than having to have someone else do all the work? If I knew what lines I had to add and where I would add my own. Maybe some documentation on how to add your own could take the pressure off. I don't 100% understand how gitub works but if I could follow some documentation, then I'm sure I could figure it out and add my own collection details into the code and even add others for people who really have no clue.

**Comment by 5ila5** [comment]
We do not hard-code collection dates but write pythons scripts obtaining the data or adding documentation for ICS sources (Only possible if the service provider offers a static link to an ICS/ICAL file). There is pretty extensive documentation on how to add a source [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/contributing.md) when following either link below `The 2 Ways of adding support for a new Service Provider`
